notebooks/huggingface_notebook.ipynb

```diff
@@ -0,0 +1,315 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install composer\n",
+    "# To install from source instead of the last release, use the below command instead:\n",
+    "# !pip install 'mosaicml[nlp]'"
```

Comment:
missing "below command" to install from source? (can probably just be omitted; we don't provide this option in most of the other notebooks)